src/request/src/middleware/src/storage/index.js

```diff
@@ -14,14 +14,14 @@ export {
   MemoryAdapter
 };
 
-export default class Storage {
+export class Storage {
```

Comment:
This is easily fixed by a good barrel file - export * from './storage'; inside index.js makes it a good barrel and you can import { Storage } from './storage'; as if it's defined inside index.js.